src/test/java/jenkins/branch/WorkspaceLocatorImplTest.java

```diff
@@ -74,11 +74,11 @@ public void locate() throws Exception {
         stuff.scheduleBuild2(0).getFuture().get();
         r.waitUntilNoActivity();
         showComputation(stuff);
-        FreeStyleProject master = r.jenkins.getItemByFullName("stuff/dev%2Fflow", FreeStyleProject.class);
+        FreeStyleProject master = r.jenkins.getItemByFullName("stuff/"+NameMangler.apply("dev/flow"), FreeStyleProject.class);
```

Comment:
Prefer to use a string constant here so a reader of the test can immediately see what the actual item name will be—and we will notice if some change to `NameMangler` potentially might be causing compatibility issues.